### PR TITLE
Adding new step to remotedeploy-combo

### PR DESCRIPTION
**Pull request by sishida**
Signed-off-by: Sarah Ishida <sishida@us.ibm.com>

PR for https://github.com/eclipse/codewind/issues/2567

### CODE REVIEW

**Comment by tetchel** [comment]
@maysunfaisal

File: docs/_documentations/remotedeploy-combo.md [hunk 1]
```diff
@@ -118,6 +115,20 @@ INFO[0159] Waiting for Codewind PFE to start
 INFO[0159] Codewind is available at: https://codewind-gatekeeper-k55333j0.mycluster-12345-7674b4bd9abbdeea5be228236d5275c9-0001.eu-gb.containers.appdomain.cloud
 ```
 
+2\. Run as priveleged and as root or install Keycloak.
```
Comment (tetchel) [inline docs/_documentations/remotedeploy-combo.md]: Typo priveleged

**Comment by tetchel** [comment]
This looks OK but i'm not certain what the distinction is between IBM Cloud public and OpenShift, so I'm probably not the right one to review. If that sectino is just copied from somewhere else it is likely fine

**Comment by tetchel** [comment]
@maysunfaisal for OpenShift isn't "project" the preferred term over "namespace"?

**Comment by maysunfaisal** [comment]
@tetchel true but looks like all other docs needs to be updated as well, I checked we have a few other places where we use namespace and project interchangeably.

@sishida do you want to do that with a new PR or with this one?

**Comment by sishida** [comment]
@maysunfaisal Let's update terminology in a new PR.

**Pull request merged**